Rollback "qa.hedgehog\hedgehog-core" dependency version from 0.6.0 back to 0.5.1 and close #4417

lmserrano opened the pull request.

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/zio/zio?pullRequest=4418) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/zio/zio?pullRequest=4418) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/zio/zio?pullRequest=4418) it.</sub>

adamgfraser commented:
@lmserrano Do we still need this? Seems like there were some specific issues with your environment and you were able to build the project without this change for your other PR?

lmserrano commented:
This change is necessary but there was another PR (#4419 ) related with some other feature but which added this in order to make it work.
Although the original issue for the dependency problem #4417 was created before the problem was reported and addressed in the mdoc PR, that PR end up being merged earlier and fixed the problem.
Thank you, @adamgfraser . I had closed the issue #4417 but the PR was still open. Closing it now too.
